native/cocos/editor-support/spine/Bone.cpp

```diff
@@ -310,193 +312,22 @@ Bone *Bone::getParent() {
     return _parent;
 }
 
-Vector<Bone *> &Bone::getChildren() {
-    return _children;
-}
-
-float Bone::getX() {
-    return _x;
-}
-
-void Bone::setX(float inValue) {
-    _x = inValue;
-}
-
-float Bone::getY() {
-    return _y;
-}
-
-void Bone::setY(float inValue) {
-    _y = inValue;
-}
-
-float Bone::getRotation() {
-    return _rotation;
-}
-
-void Bone::setRotation(float inValue) {
-    _rotation = inValue;
-}
-
-float Bone::getScaleX() {
-    return _scaleX;
-}
-
-void Bone::setScaleX(float inValue) {
-    _scaleX = inValue;
-}
-
-float Bone::getScaleY() {
-    return _scaleY;
-}
-
-void Bone::setScaleY(float inValue) {
-    _scaleY = inValue;
-}
-
-float Bone::getShearX() {
-    return _shearX;
-}
-
-void Bone::setShearX(float inValue) {
-    _shearX = inValue;
-}
-
-float Bone::getShearY() {
-    return _shearY;
-}
-
-void Bone::setShearY(float inValue) {
-    _shearY = inValue;
-}
-
-float Bone::getAppliedRotation() {
-    return _arotation;
-}
-
-void Bone::setAppliedRotation(float inValue) {
-    _arotation = inValue;
-}
-
-float Bone::getAX() {
-    return _ax;
-}
-
-void Bone::setAX(float inValue) {
-    _ax = inValue;
-}
-
-float Bone::getAY() {
-    return _ay;
-}
-
-void Bone::setAY(float inValue) {
-    _ay = inValue;
-}
-
-float Bone::getAScaleX() {
-    return _ascaleX;
-}
-
-void Bone::setAScaleX(float inValue) {
-    _ascaleX = inValue;
-}
-
-float Bone::getAScaleY() {
-    return _ascaleY;
-}
-
-void Bone::setAScaleY(float inValue) {
-    _ascaleY = inValue;
-}
-
-float Bone::getAShearX() {
-    return _ashearX;
-}
-
-void Bone::setAShearX(float inValue) {
-    _ashearX = inValue;
-}
-
-float Bone::getAShearY() {
-    return _ashearY;
-}
-
-void Bone::setAShearY(float inValue) {
-    _ashearY = inValue;
-}
-
-float Bone::getA() {
-    return _a;
-}
-
-void Bone::setA(float inValue) {
-    _a = inValue;
-}
-
-float Bone::getB() {
-    return _b;
-}
-
-void Bone::setB(float inValue) {
-    _b = inValue;
-}
-
-float Bone::getC() {
-    return _c;
-}
-
-void Bone::setC(float inValue) {
-    _c = inValue;
-}
-
-float Bone::getD() {
-    return _d;
-}
-
-void Bone::setD(float inValue) {
-    _d = inValue;
-}
-
-float Bone::getWorldX() {
-    return _worldX;
-}
-
-void Bone::setWorldX(float inValue) {
-    _worldX = inValue;
-}
-
-float Bone::getWorldY() {
-    return _worldY;
-}
-
-void Bone::setWorldY(float inValue) {
-    _worldY = inValue;
```

Comment:
Move the implementation of getter/setter to header and inline them.
Inlining them could make wasm/asmjs package size smaller and improve performance a little bit.